src/sentry/static/sentry/app/components/fileChange.tsx

```diff
@@ -0,0 +1,51 @@
+import PropTypes from 'prop-types';
+import React from 'react';
+import styled from '@emotion/styled';
+
+import space from 'app/styles/space';
+import overflowEllipsis from 'app/styles/overflowEllipsis';
+import {AvatarUser} from 'app/types';
+import AvatarList from 'app/components/avatar/avatarList';
+import {IconFile} from 'app/icons';
+
+type Props = {
+  filename: string;
+  authors: AvatarUser[];
+};
+
+function FileChange({filename, authors}: Props) {
+  return (
+    <FileItem className="list-group-item list-group-item-sm">
+      <Filename>
+        <StyledIconFile size="xs" />
+        {filename}
+      </Filename>
+      <div>
```

Comment:
is this `div` really necessary? I believe you need it because of` justify-content: space-between;` (2x items)..right?

Reply:
It might not be needed. I'll see if I can remove it.